### ISSUE TITLE
Multiplayer capabilities

### ISSUE THREAD

**Issue by SeventhSonOfASeventhSon** [marcusbuffett/command-line-chess#7]
I think that it would be really good if this Chess app had multiplayer capabilities so that one could play with someone else who has this app, or it could even just be people on the same network with the app installed, but whatever it is, some sort of multiplayer capabilities would be good.

**Comment by marcusbuffett** [comment]
Agreed that would be very cool, wouldn't be overly hard to write, I'll look into it.

**Comment by jeromew21** [comment]
Interested in this still?

**Comment by marcusbuffett** [comment]
For sure, multiplayer would be great. Wonder if it would be possible to use this as a front end for an established chess platform (chess.com, lichess, etc). Could make this task significantly easier.

**Comment by jeromew21** [comment]
Unfortunately lichess and Chess.com don't have live chess API's as far as I know. I've started to work on adding functionality to play on the same network (and presumably not only limited to that, if you do port forwarding)

**Comment by Kabouik** [comment]
I think lichess now has an API, and a cli tool was posted in another issue here (`lichess-cli`) and allows resuming ongoing games. There's a more official `lichs` too but as far as I understand it only allows matchmaking. It would be great if `cl-chess` could make use of that API and include one of those features (or both) to have a common interface for single player and multiplayer.
